tests/e2e/testcases/subnet/sov/addRemoveValidatorPoS/suite.go

```diff
@@ -177,153 +177,3 @@ var _ = ginkgo.Describe("[AddRemove Validator SOV L1 Manager PoS]", func() {
 		commands.DeleteSubnetConfig(utils.BlockchainName)
 	})
 })
-
-var _ = ginkgo.Describe("[AddRemove Validator SOV External Manager PoS]", func() {
```

Comment:
why are we removing this

Reply:
sov on c-chain stopped working as external manager is on c-chain which does not have native minter precompile. it was working before because previously there was no reward paiment so the precompile issue was not hit